cron/centAcl-Func.php

```diff
@@ -128,36 +128,35 @@ function getCentAclRunningState()
  * @param integer $res_id
  * @return array $host
  */
-function getFilteredPollers($host, $res_id)
+function getFilteredPollers($host, $resId)
 {
-    global $pearDB, $hostCache;
+    global $pearDB;
 
     $dbResult = $pearDB->prepare(
         "SELECT COUNT(*) AS count FROM acl_resources_poller_relations WHERE acl_res_id = :resId"
     );
-    $dbResult->bindValue(':resId', $res_id, \PDO::PARAM_INT);
+    $dbResult->bindValue(':resId', $resId, \PDO::PARAM_INT);
     $dbResult->execute();
-
     $row = $dbResult->fetch();
     $isPollerFilter = $row['count'];
 
     $hostTmp = $host;
     $dbResult = $pearDB->prepare(
-        "SELECT host_host_id " .
-        "FROM acl_resources_poller_relations, acl_resources, ns_host_relation " .
-        "WHERE acl_resources_poller_relations.acl_res_id = acl_resources.acl_res_id " .
-        "AND acl_resources.acl_res_id = :resId " .
-        "AND ns_host_relation.nagios_server_id = acl_resources_poller_relations.poller_id " .
-        "AND acl_res_activate = '1'"
+        "SELECT host_host_id
+        FROM acl_resources_poller_relations, acl_resources, ns_host_relation
+        WHERE acl_resources_poller_relations.acl_res_id = acl_resources.acl_res_id
+        AND acl_resources.acl_res_id = :resId
+        AND ns_host_relation.nagios_server_id = acl_resources_poller_relations.poller_id
+        AND acl_res_activate = '1'"
     );
-    $dbResult->bindValue(':resId', $res_id, \PDO::PARAM_INT);
+    $dbResult->bindValue(':resId', $resId, \PDO::PARAM_INT);
     $dbResult->execute();
 
     if ($dbResult->rowCount()) {
         $host = array();
         while ($row = $dbResult->fetch()) {
             if (isset($hostTmp[$row['host_host_id']])) {
-                $host[$row['host_host_id']] = $hostCache[$row['host_host_id']];
+                $host[$row['host_host_id']] = 1;
```

Comment:
Why you fix the value "1"?

Reply:
I guess cause isset is better (in performance) than in_array